packages/paste-website/src/pages/components/disclosure/index.mdx

```diff
@@ -430,19 +459,69 @@ const PopoverExample: React.FC = () => {
 
 #### Props
 
+#### useDisclosureState Props
+
+Pass these as part of an object to the useDisclosureState hook.
+
+##### `baseId?: string`
+
+ID that will serve as a base for all the items IDs.
+
+##### `visible?: boolean`
+
+Whether it's visible or not.
+
+##### `animated?: number | boolean`
+
+If true, animating will be set to true when visible is updated. It'll wait
+for stopAnimation to be called or a CSS transition ends. If animated is set
+to a number, stopAnimation will be called only after the same number of
+milliseconds have passed.
+
+#### useDisclosureState returned props
+
+These props are returned by the state hook. You can spread them into this component (`{...state}`) or pass them separately. You can also provide these props from your own state logic.
+
+##### `baseId?: string`
+
+ID that will serve as a base for all the items IDs.
+
+##### `visible?: boolean`
+
+Whether it's visible or not.
+
+##### `animated?: number | boolean`
+
+If true, animating will be set to true when visible is updated. It'll wait
+for stopAnimation to be called or a CSS transition ends. If animated is set
+to a number, stopAnimation will be called only after the same number of
+milliseconds have passed.
+
+##### `animating?: boolean`
+
+Whether it's animating or not.
```

Comment:
Does this work with our animations?

Reply:
Just verified. It doesn't since we're using our animation library.